apps/api/v2/src/modules/oauth-clients/oauth-client.repository.ts

```diff
@@ -15,11 +16,14 @@ export class OAuthClientRepository {
   ) {}
 
   async createOAuthClient(organizationId: number, data: CreateOAuthClientInput) {
+    const id = "cal_pla_" + randomBytes(28).toString("hex");
+
     return this.dbWrite.prisma.platformOAuthClient.create({
       data: {
         ...data,
         secret: this.jwtService.sign(data),
         organizationId,
+        id,
```

Comment:
wondering the reason behind creating the id manually ?

Reply:
Was having issues for some reason with Prisma, will revert.